TestFoundation/TestNSURL.swift

```diff
@@ -417,6 +418,16 @@ class TestNSURL : XCTestCase {
             XCTAssertEqual(result, "file:///tmp/")
         }
     }
+
+    func test_copy() {
+        let url = NSURL(string: "https://www.swift.org")
+        let urlCopy = url!.copy() as! NSURL
+        XCTAssertTrue(url === urlCopy)
```

Comment:
This requires that the url instance is the same as the other; we may change that in the future. I would prefer the test checks a property of the URL for equality, or even better calls into URL's isEqual